test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -157,7 +158,7 @@ TEST_F(ClusterManagerImplTest, MultipleProtocolClusterFail) {
       http_protocol_options: {}
   )EOF";
   EXPECT_THROW_WITH_MESSAGE(
-      create(parseBootstrapFromV3Yaml(yaml)), EnvoyException,
+      create(parseBootstrapFromV3Yaml(yaml, false)), EnvoyException,
```

Comment:
Since this isn't a deprecated test, how come we don't avoid boosting? Ideally we update all tests to v3 that aren't deprecated.